library/core-components/components/nav/nav.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { Meta, StoryObj } from '@storybook/react';
+import {
+  Github,
+  Figma,
+  EventNote,
+  Instagram,
+  LinkedIn,
+  Twitter,
+  Youtube,
+} from '@rangle/radius-foundations/generated/icons';
+
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
+import { RadiusNav } from './nav';
+
+import RangleLogoLight from '../../../../radius/assets/rangle-logo-light.svg';
+import RadiusLogoLight from '../../../../radius/assets/radius-logo-light.svg';
+import { radiusTokens } from '@rangle/radius-foundations/generated/design-tokens.constants';
+
+const meta: Meta<typeof RadiusNav> = {
+  component: RadiusNav,
+  title: 'Nav',
+  parameters: {
+    badges: [BADGE.EXPERIMENTAL],
+  },
+  argTypes: {
+    as: {
+      control: {
+        type: 'text',
+      },
+      table: {
+        defaultValue: { summary: 'nav' },
+      },
+    },
+    className: { table: { disable: true } },
+    ref: { table: { disable: true } },
+  },
+  args: {
+    // TODO: make image src dynamic based on token (WIP)
+    // TODO: apply height to all logos as children?
+    logos: (
+      <>
+        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+        <a href="#">
+          <img
+            src={RangleLogoLight}
+            alt="Rangle.io logo"
+            style={{
+              height: `var(${radiusTokens.component.sizing.navigation.secondaryLogo})`,
+            }}
+          />
+        </a>
+        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+        <a href="#">
+          <img
+            src={RadiusLogoLight}
+            alt="Radius logo"
+            style={{
+              height: `var(${radiusTokens.component.sizing.navigation.primaryLogo})`,
+            }}
+          />
+        </a>
+      </>
+    ),
```

Comment:
We still need to find a good solution for dynamic themeable images. We may end up having to use an `emotion` theme provider and handling it via JS